### ISSUE TITLE
 CNTools v7.0.0 alpha displays wrong owner names in offline tx file

### ISSUE THREAD

**Issue by amani1104** [cardano-community/guild-operators#693]
**Describe the bug**
When initiating owner updates with the online client, CNTools creates an offline tx file that is to be signed with the secret keys in the offline client. However, the owner entries are not assigned correctly in the tx file and always refer to the HW owner in the owner set. Below is an example with two owners, but I also tested with more owners, which all are assigned the name of the HW owner.  

```
Do you want to sign Pool Update with: Owner #1 'letzbake' payment signing key ?
Selected value: [y] Yes

Do you want to sign Pool Update with: Owner #1 'letzbake' stake signing key ?
Selected value: [y] Yes

Do you want to sign Pool Update with: Owner #2 'letzbake' stake signing key ?
```

**To Reproduce**
1. Update pool with multiple owners with the online client of which one needs to be a HW owner 
2. Generate tx file
3. See error

**Expected behavior**
Owner names are correctly assigned with respect to wallet folder names.

**Version:**
 - OS: Ubuntu 20.04 LTS
 - Product version: CNTools 7.0.0. alpha
 - Cardano Node version: 1.24.2
 - Network you're connecting to: Mainnet

**Comment by Scitz0** [comment]
Solved in #694 soon to be merged into alpha

**Comment by Scitz0** [comment]
Fix merged into alpha.
